.gitignore

```diff
@@ -9,3 +9,15 @@ venv/
 *.pyc
 app/db.sqlite3
 db.sqlite3
+manage.py
+taxi_service_db_data.json
+taxi_service/__init__.py
+taxi_service/asgi.py
+taxi_service/wsgi.py
+taxi/migrations
+taxi/__init__.py
+taxi/admin.py
+taxi/apps.py
+taxi/models.py
+taxi/tests.py
+.gitignore
```

Comment:
You shouldn't ignore models, manage.py and other files. You should think like: "If I would clone this project, do I need those files to be there? Will it work properly?" In this case, you don't want to ignore manage.py, init files ect.
Also ignoring gitignore doesn't work 👀
Check [Gitignore for Django project recommendations](https://djangowaves.com/tips-tricks/gitignore-for-a-django-project/) or google other info